docs/account-project-management/allocation-management/allocation-management.md

```diff
@@ -88,9 +88,9 @@ PI adds hours to a suballocation:
 sbank e sub <projectname>::<nameofsuballoc> --hours-to-move <hours> --to-suballocation <projectname>::<nameofsuballoc2>
 ```
 
-!!! note 
+!!! note
 
-    `hours-to-move` must be greater than or equal to the available balance for the suballocation `nameofsuballoc`.
+    `hours-to-move` must be less than or equal to the available balance for the suballocation `nameofsuballoc`.
```

Comment:
it took me a few tries when reading this to understand it, since the source and destination are referred to as suballocations. Maybe would help to use different variable names?

Reply:
```bash
sbank e sub <projectname>::<source-suballoc> --hours-to-move <hours> --to-suballocation <projectname>::<dest-suballoc>

Reply:
up to you, feel free to merge